frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -85,6 +88,18 @@ local footerTextGeneratorMap = {
         return footer:getDataFromStatistics(
             "TC: ", (left and left or footer.pages - footer.pageno))
     end,
+    mem_usage = function(footer)
+        local statm = io.open("/proc/self/statm", "r")
+        if statm then
+            local infos = statm:read("*all")
+            statm:close()
+            local rss = infos:match("^%S+ (%S+) ")
```

Comment:
It means "Resident size", i didn't make that up :)
It's the 2nd of the 7 fields you get from `/proc/<pid or self>/statm:`
https://www.kernel.org/doc/Documentation/filesystems/proc.txt
http://man7.org/linux/man-pages/man5/proc.5.html
```
       /proc/[pid]/statm
              Provides information about memory usage, measured in pages.
              The columns are:

                  size       (1) total program size
                             (same as VmSize in /proc/[pid]/status)
                  resident   (2) resident set size
                             (same as VmRSS in /proc/[pid]/status)
                  shared     (3) number of resident shared pages (i.e., backed by a file)
                             (same as RssFile+RssShmem in /proc/[pid]/status)
                  text       (4) text (code)
                  lib        (5) library (unused since Linux 2.6; always 0)
                  data       (6) data + stack
                  dt         (7) dirty pages (unused since Linux 2.6; always 0)
```
RSS is the common name for that, that you see in headers of `ps auxw` on linux:
```
USER       PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
me       31098  1.5  1.4 318696 58024 pts/4    S+   15:03   0:01 ./luajit ./reader.lua -d test
```